### DISCUSSION TITLE
Units in mfem

### DISCUSSION THREAD

**Discussion by moroz3339** [mfem/mfem#3212] [Q&A] [answered]
Hello everyone!
I am trying to solve the linear elasticity problem by mfem. I follow examples/ex2.cpp, but validation of result shows me that my result is wrong. I use force in Newtons and mesh in metres. As I think gotten displacement should be also in metres. It is true or mfem makes implicitly transformation of mesh or border condition?
Thanks!

**Comment by prathamgit** [comment, marked as answer]
As far as I know, mfem does not assume or change any particular units in the simulation. The output units depend on the supplied mesh units (not specifying any units, but just interpreting a vertex (1.0, 0.5) as (1.0 m, 0.5 m)) and any constants and boundary conditions that are supplied by the user. It is all about how we interpret the numbers processed by the calculator called mfem.
I believe you must be supplying non-SI units in one of your constants or must have made any other mistake in the code.

**Reply by moroz3339** [reply]
Thank you for your answer!
